### ISSUE TITLE
Modify basic page config to use layout builder by default

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa01#92]
Per #87, this is a separate task than updating the collegiate_basic profile.

**Comment by pyrello** [comment]
For this to work, we'll need to abstract out a "Core" module that includes the field storage definition for layout builder, unless we can set a unique field name without breaking everything.

**Comment by bspeare** [comment]
This might already be on your roadmap, but Breadcrumbs and Page Title will also need to be removed from the block configuration and moved to a layout section by default.

**Comment by pyrello** [comment]
Dependent on #104
